code-scanning/codeql.yml

```diff
@@ -40,11 +40,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ $detected-codeql-languages ]
-        # CodeQL supports [ $supported-codeql-languages ]
-        # Use only 'java-kotlin' to analyze code written in Java, Kotlin or both
-        # Use only 'javascript-typescript' to analyze code written in JavaScript, TypeScript or both
-        # Learn more about CodeQL language support at https://aka.ms/codeql-docs/language-support
+        $codeql-languages-matrix
+        # CodeQL supports the following values keywords for 'language': $supported-codeql-languages
+        # Use 'java-kotlin' to analyze code written in Java, Kotlin or both
+        # Use 'javascript-typescript' to analyze code written in JavaScript, TypeScript or both
+        # To learn more about changing the languages that are analyzed or customizing the build mode for your analysis,
+        # see https://docs.github.com/en/code-security/code-scanning/creating-an-advanced-setup-for-code-scanning/customizing-your-advanced-setup-for-code-scanning.
```

Comment:
Could we convert this to a shortlink to give us flexibility to change this docs URL in the future?  Also we should probably hold off on merging this change until the docs updates are complete.

Reply:
> Could we convert this to a shortlink to give us flexibility to change this docs URL in the future?

We are not doing this consistently for the other links. I also think the links here should probably be considered valid when committed, and it would be odd if we change what they are pointing to.
If we know that we are going to change them, let's go with the correct ones.

> Also we should probably hold off on merging this change until the docs updates are complete.

good point.

Reply:
Merged CodeQL from action/workflow into SHA f4dbb8424c
Branch: main
965022-oss-patch3